app/Console/Commands/RebuildQueryserviceData.php

```diff
@@ -15,7 +15,7 @@
 class RebuildQueryserviceData extends Command {
     use Traits\PageFetcher;
 
-    protected $signature = 'wbs-qs:rebuild {--domain=*} {--chunkSize=50} {--queueName=manual-intervention} {--sparqlUrlFormat=http://queryservice.default.svc.cluster.local:9999/bigdata/namespace/%s/sparql}';
+    protected $signature = 'wbs-qs:rebuild {--domain=*} {--chunkSize=50} {--queueName=default} {--sparqlUrlFormat=http://queryservice.default.svc.cluster.local:9999/bigdata/namespace/%s/sparql}';
```

Comment:
praise: yeah, excellent spot and good prediction from 2 years ago that this would be a thing some fool (aka. Tom) would trip up on!